include/boost/url/bnf/parse.hpp

```diff
@@ -77,214 +96,182 @@ parse(
     error_code& ec,
     char ch) noexcept;
 
-/** @brief Parse a sequence of elements
+/** Parse beginning of a char range as a sequence of arbitrary BNF elements
```

Comment:
I usually make the brief (first line of javadoc) generic. For example "Parse a substring into one or more BNF elements"

Reply:
OK. Changing it back.

There are two components in that. For the second component, "one or more BNF elements" and "one or more BNF elements" are almost synonymous. 

As for the first component in "Parse a substring into one or more BNF elements" is similar to something I had before (but after "Parse a sequence of elements"). The reason I changed it was because it looked ambiguous when I compared it with other overloads. For instance, "Parse a substring into one or more BNF elements" could also apply to `parse_string`, because a `string_view` is also potentially a substring. 

As both a string_view and two pointers represent a subrange/substring, it seemed like the relevant difference between these overloads was whether they consider the whole sequence or only the beginning of the sequence.

I'm changing it back but, just in case, that was the rationale.